inc/customizer/options/base_layout_single.php

```diff
@@ -0,0 +1,553 @@
+						'cssVar' => [
+							'vars'       => [
+								'--textAlign',
+								'--justify',
+							],
+							'valueRemap' => [
+								'--justify' => [
+									'left'   => 'flex-start',
+									'center' => 'center',
+									'right'  => 'flex-end',
+								],
+							],
+							'responsive' => true,
+							'selector'   => $this->cover_selector . ',' . $this->cover_selector . ' .container, ' . ( $this->post_type === 'post' ? '.entry-header' : '.nv-page-title-wrap' ),
+						],
+					],
+					'active_callback'       => $this->post_type === 'post' ? '__return_true' : function() {
+						return ! get_theme_mod( 'neve_page_hide_title', false );
+					},
+				],
+				'\Neve\Customizer\Controls\React\Responsive_Radio_Buttons'
+			)
+		);
+
+		$this->add_control(
+			new Control(
+				'neve_' . $this->post_type . '_title_position',
+				[
+					'sanitize_callback' => 'neve_sanitize_position',
+					'transport'         => $this->selective_refresh,
+					'default'           => [
+						'mobile'  => 'center',
+						'tablet'  => 'center',
+						'desktop' => 'center',
+					],
+				],
+				[
+					'label'                 => esc_html__( 'Title Position', 'neve' ),
+					'section'               => $this->section,
+					'priority'              => $this->post_type === 'post' ? 6 : 35,
+					'choices'               => [
+						'flex-start' => [
+							'tooltip' => esc_html__( 'Top', 'neve' ),
+							'icon'    => 'arrow-up',
+						],
+						'center'     => [
+							'tooltip' => esc_html__( 'Middle', 'neve' ),
+							'icon'    => 'sort',
+						],
+						'flex-end'   => [
+							'tooltip' => esc_html__( 'Bottom', 'neve' ),
+							'icon'    => 'arrow-down',
+						],
+					],
+					'live_refresh_selector' => true,
+					'live_refresh_css_prop' => [
+						'cssVar' => [
+							'vars'       => '--vAlign',
+							'responsive' => true,
+							'selector'   => $this->cover_selector,
+						],
+					],
+					'show_labels'           => true,
+					'active_callback'       => function() {
+						return $this->post_type === 'post' ? $this->is_cover_layout() : $this->is_cover_layout() && ! get_theme_mod( 'neve_page_hide_title', false );
+					},
+				],
+				'\Neve\Customizer\Controls\React\Responsive_Radio_Buttons'
+			)
+		);
+
+		// This control could be added in layout_single_post but because all controls there need to have priority = 6, its position will be wrong
+		if ( $this->post_type === 'post' ) {
+			$this->add_control(
+				new Control(
+					'neve_post_cover_meta_before_title',
+					[
+						'sanitize_callback' => 'neve_sanitize_checkbox',
+						'default'           => false,
+					],
+					[
+						'label'           => esc_html__( 'Display meta before title', 'neve' ),
+						'section'         => $this->section,
+						'type'            => 'neve_toggle_control',
+						'priority'        => 6,
+						'active_callback' => [ $this, 'is_cover_layout' ],
+					],
+					'Neve\Customizer\Controls\Checkbox'
+				)
+			);
+		}
```

Comment:
@preda-bogdan it's a mix-up with some pro settings. When this was developed, to avoid changing some priorities for controls in PRO, I had to put all the controls with priority 6 so when the pro version is adding some controls, it will add after the lite settings. Having this in another file means that it will lose its position due to the fact that when two controls are having the same priority, they will be displayed in the order they were added in the code.

Reply:
@cristian-ungureanu Ok, I understand, but couldn't we have had a hook that PRO would use to register additional controls in the correct order? Would this be maybe possible in the future just to keep the code clean and minimise hacks if they can be avoided?

Reply:
Nvm, I changed the priorities in PRO and moved the function to its file.